Handling error when perunPrincipal.user is null

DanoFecko opened the pull request:
Resolves #113 

* sorry for double commit, it was a lot easier to keep to old one here

Vojtech-Sassmann commented on libs/perun/services/src/lib/init-auth.service.ts:
```diff
@@ -40,8 +43,12 @@ export class InitAuthService {
-        this.storeService.setPerunPrincipal(perunPrincipal);
-        this.authResolver.init(perunPrincipal);
+        if (perunPrincipal.user === null) {
+          this.dialog.open(UserDontExistDialogComponent, {height: '80px'});
```
Why is there a height limit? This height causes a scrollbar to appear.

DanoFecko replied:
without it, dialog had lot of empty space under text. It didn't show scrollbar on my screen, but i didn't check smaller resolutions.

DanoFecko replied:
i will play a bit with it and find solution

Vojtech-Sassmann replied:
To reduce the padding or margin of an element, just change the CSS. Or, you can even create a new class with only those CSS attributes that you want to use.

Vojtech-Sassmann replied:
This should not be hard at all. It is just a matter of a couple of minutes to write a CSS class and apply it.

DanoFecko commented:
@Vojtech-Sassmann I fixed by placing the text to the dialog content instead of title. I also disabled closing the dialog by clicking on the background and somehow I had .css file in component instead of .scss, so I changed that too.
